FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Andreas Link on 13.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
As per my previous comments, the `internal` keyword should be omitted.
Also, this class is not supposed to be subclassed, so it should be marked as final.
```suggestion
final class ManagedCache: NSManagedObject {
```